renderer/containers/cropper.js

```diff
@@ -123,6 +123,16 @@ export default class CropperContainer extends Container {
     this.setState(updates);
   }
 
+  setSize = ({width: defaultWidth, height: defaultHeight}) => {
+    let {width, height} = this.state;
```

Comment:
```suggestion
    const {width = defaultWidth, height = defaultHeight} = this.state;
```

Reply:
Are you sure? In this case every time you click on crop area button, it will set the default size. For example, if user set size, return to main menu and click  crop button again, it will reset his settings to default.

Reply:
I'm not sure. It was just a suggestion when looking at the code without running it.